README.md

```
@@ -0,0 +1,780 @@
+        ├── models.py
+        ├── serializer.py
+        ├── tests.py
+        ├── urls.py
+        ├── utils
+        └── views.py
+```
+
+### Project Index
+
+<details open>
+	<summary><b><code>DREAMSERVICE/</code></b></summary>
```

markdownlint-cli2 (0.18.1)

Hard tabs
Column: 1

(MD010, no-hard-tabs)